rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -776,35 +776,35 @@ void spu_thread::do_dma_transfer(const spu_mfc_cmd& args)
 		{
 			auto& res = vm::reservation_lock(eal, 1);
 			*static_cast<u8*>(dst) = *static_cast<const u8*>(src);
-			res &= ~1ull;
+			res++;
```

Comment:
if putllc is used on memory that was written by PUT cmd on another spu (assume there was a reservation there)
it will detect reservation failure on the first timestamp check instead of also doing 128 bytes check. 
so the apllication will restart atomic loop a little faster after such failure because of it.

Reply:
Is it practically relevant, or just a speculation? I'd like to avoid inconsistency with TSX path if possible. Plain writes in reservation area are limited in practice.

Reply:
It is technically an improvement, it will be too late to change it now as its barried under quite a few code conflicts.
Those lines were modified a few times in this PR.